projects/ion/src/lib/input-counter/input-counter.component.scss

```
@@ -5,12 +5,22 @@
   border: 1px solid $neutral-5;
   border-radius: 6px;
```

```suggestion
  border-radius: 9px;
```
When hovering it is showing a white space between the button and the container border